src/commands/BootstrapCommand.js

```diff
@@ -92,12 +97,32 @@ export default class BootstrapCommand extends Command {
   }
 
   /**
-   * Run the "prepublish" NPM script in all bootstrapped packages
+   * Run the "prepublish" or "prepare" (depending on the version of npm client)
+   * NPM script in all bootstrapped packages
    * @param callback
    */
   prepublishPackages(callback) {
     this.logger.info("Prepublishing packages");
-    this.runScriptInPackages("prepublish", callback);
+
+    const npmClientVersionString = NpmUtilities.getNpmClientVersion();
+    const npmClientMajorVersion = parseInt(npmClientVersionString, 10);
+
+    if (npmClientMajorVersion <= 3) {
```

Comment:
The reason behind this check was discussed [here](https://github.com/lerna/lerna/issues/421#issuecomment-267119633). We, actually, just want to behave the same as your current instance of npm.